dev/tasks/homebrew-formulae/autobrew/apache-arrow.rb

```diff
@@ -19,7 +19,7 @@
 class ApacheArrow < Formula
   desc "Columnar in-memory analytics layer designed to accelerate big data"
   homepage "https://arrow.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=arrow/arrow-0.17.0.9000/apache-arrow-0.17.0.9000.tar.gz"
+  url "https://www.apache.org/dyn/closer.lua?path=arrow/arrow-0.17.1.9000/apache-arrow-0.17.1.9000.tar.gz"
   sha256 "9948ddb6d4798b51552d0dca3252dd6e3a7d0f9702714fc6f5a1b59397ce1d28"
```

Comment:
I suspect you'll need to change the sha signature. Why don't we see a failure, is this going to fail in the nightly crossbow build?

Reply:
Nope. That URL doesn't exist; it's just to keep the version numbers in sync. We always install `--HEAD` in our tests.